account_due_list/views/payment_view.xml

```diff
@@ -231,7 +234,8 @@
         <field name="view_mode">tree,form,pivot,graph</field>
         <field
             name="context"
-        >{'search_default_posted_moves': 1, 'search_default_from_invoices': 1, 'search_default_not_full_reconciled': 1}</field>
+        >{'search_default_posted_moves': 1, 'search_default_from_invoices': 1,
```

Comment:
Weird that pre-commit is doing this split when previously it was not done nor failed...